src/proto/Stellar.proto

```diff
@@ -37,6 +37,38 @@ message OperationChangeTrust {
     int64 valid_before = 2;
 }
 
+enum ClaimPredicate {
+    Claim_predicate_unconditional = 0;
```

Comment:
```suggestion
    Unconditional = 0;
```

Reply:
These are in the Stellar namespace, so prefix is desirable. I've shortened the prefix.